pkgs/by-name/au/authentik/package.nix

```
@@ -0,0 +1,248 @@
+
+    postPatch = ''
+      cd web
+    '';
+
+    installPhase = ''
+      mkdir $out
+      cp -r dist $out/dist
+      cp -r authentik $out/authentik
+    '';
+
+    NODE_ENV = "production";
+    NODE_OPTIONS = "--openssl-legacy-provider";
+
+    npmInstallFlags = [ "--include=dev" ];
+  };
+
+  python = python3.override {
+    packageOverrides = final: prev: {
+      authentik-django = final.buildPythonPackage {
+        pname = "authentik-django";
+        inherit version src;
+        pyproject = true;
+
+        postPatch = ''
+          substituteInPlace authentik/root/settings.py \
+            --replace 'Path(__file__).absolute().parent.parent.parent' "\"$out\""
+          substituteInPlace authentik/lib/default.yml \
+            --replace '/blueprints' "$out/blueprints"
+        '';
+
+        nativeBuildInputs = [ final.poetry-core ];
+        propagatedBuildInputs = with final; [
+          argon2-cffi
+          celery
+          channels
+          channels-redis
+          colorama
+          dacite
+          daphne
+          deepmerge
+          defusedxml
+          django
+          django-filter
+          django-guardian
+          django-model-utils
+          django-prometheus
+          django-redis
+          djangorestframework
+          djangorestframework-guardian2
+          docker
+          drf-spectacular
+          duo-client
+          facebook-sdk
+          flower
+          geoip2
+          gunicorn
+          httptools
+          kubernetes
+          ldap3
+          lxml
+          opencontainers
+          packaging
+          paramiko
+          psycopg
+          pycryptodome
+          pydantic
+          pydantic-scim
+          pyjwt
+          pyyaml
+          requests-oauthlib
+          sentry-sdk
+          structlog
+          swagger-spec-validator
+          twilio
+          twisted
+          ua-parser
+          urllib3
+          uvicorn
+          uvloop
+          watchdog
+          webauthn
+          websockets
+          wsproto
+          xmlsec
+          zxcvbn
+          jsonpatch
+        ] ++ [
+          codespell
+        ];
+
+        postInstall = ''
+          mkdir -p $out/web $out/website
+          cp -r lifecycle manage.py $out/${final.python.sitePackages}/
+          cp -r blueprints $out/
+          cp -r ${webui}/dist ${webui}/authentik $out/web/
+          cp -r ${website} $out/website/help
+          ln -s $out/${final.python.sitePackages}/lifecycle $out/lifecycle
+        '';
+      };
+    };
+  };
+
+  authentik-django = python.pkgs.authentik-django;
+
+  proxy = buildGoModule {
+    pname = "authentik-proxy";
+    inherit version;
+
+    src = runCommand "authentik-proxy-source" {} ''
+      mkdir -p $out/web $out/authentik
+
+      cp -r ${src}/cmd ${src}/internal $out/
+      cp -r ${src}/authentik/lib $out/authentik/
+      cp ${src}/web/static.go $out/web/
+
+      cp ${src}/web/robots.txt ${src}/web/security.txt $out/web/
```

```suggestion
      cp ${src}/web/* $out/web/
```

Any reason to not copy everything?